bin/normalize-ways

```diff
@@ -5,6 +5,7 @@
 var byline = require('byline');
 var fs = require('graceful-fs');
 var path = require('path');
+var geojsonLR = require('@mapbox/geojson-lr');
```

Comment:
@nickcordella you're adding a closed source dependency to an open source library ;)